### PR TITLE
[pythonic resources] update example with_great_expectations

### PR DESCRIPTION
**Pull request by yuhan**
## How I Tested These Changes
`dagster dev` loads and run is green

### CODE REVIEW

**Comment by yuhan** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13610** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13610" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13610?utm_source=stack-comment).

File: examples/with_great_expectations/with_great_expectations/ge_demo.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 from pandas import read_csv
 
 
+class GEOpConfig(Config):
+    csv_path: str = file_relative_path(__file__, "./data/succeed.csv")
+
+
 @op
 def read_in_datafile(csv_path):
     return read_csv(csv_path)
```
Comment (jamiedemaria) [inline examples/with_great_expectations/with_great_expectations/ge_demo.py:13]: maybe this is a feature of pydantic config i missed, but wouldn't this need to be 
```
@op
def read_in_datafile(config: GEOpConfig):
    return read_csv(config.csv_path)
```

Comment (yuhan) [reply examples/with_great_expectations/with_great_expectations/ge_demo.py:13]: oops. was just testing while i opened this pr 😓

**Review by jamiedemaria — APPROVED** [review]
nice!

**Pull request merged**